portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Raul's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="topbar">
+      <ul>
+          <li class="topbartext">Raul Cruz</li>
+          <li class="logo"><a href="https://github.com/raulcruise" target="_blank"><img src="/images/github.png" alt="GitHub logo" style="right:25px;"></a></li>
+          <li class="logo"><a href="https://www.linkedin.com/in/raulcruise/" target="_blank"><img src="/images/linkedin.png" alt="LinkedIn logo" style="right:60px;"></a></li>
+      </ul>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="border-bottom:3px solid whitesmoke">About Me</h1>
```

Comment:
I think we prefer to use CSS for this rather than inline styles.

Reply:
Done.